adapters/outbrain/outbraintest/supplemental/optional_params.json

```diff
@@ -26,6 +27,8 @@
         }
       }
     ],
+    "bcat": ["should-be-overwritten"],
```

Comment:
Not sure why are `bcat` and `badv` values are same as `tagid` in this test.. If the intention was to just assign these some string values and have nothing to do with `tagid`, I'd suggest replacing it with some other string values so that it's not confusing with the `tagid` having the same value